DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
 
-This method will be called by the app code when the true[X] unit is ready to be displayed to the user. This can be called anytime after the unit is initialized.
+The parameters for this method call are:
 
-The app should have as much extraneous UI hidden as possible, including player controls, status bars and soft buttons/keyboards, where possible.
+* `baseViewController`: The `viewController` that `TruexAdRenderer` should present its `ViewController` to. The `baseViewController` should cover the entire screen.
 
-Once `start()` is called, the renderer will wait for the ad request triggered in `init()` to finish.
 
-If the request returns an ad, the renderer will immediately instantiate a `ViewController` and present it on the `baseViewController`. Once this is complete, it will fire the `AD_STARTED` event. The user will be shown the true[X] choice card once all relevant assets have loaded. `AD_FREE_POD` and `AD_COMPLETED` may fire after this point, depending on the user's choices.
+#### `stop`
 
-If the request returns no ads, the renderer will fire the `NO_ADS_AVAILABLE` event.
+```objective-c
+    - (void)stop;
+```
 
-If the request signals that the user is in an ad-free state, then the renderer will immediately instantiate a `ViewController` and present it on the `baseViewController`. Once this is complete, it will fire the `AD_STARTED` event. After 3 seconds of a "skip card" being shown to the user, the `AD_FREE_POD` event will fire, followed immediately by the `AD_COMPLETED` event.
+The `stop` method is only called when the app needs the renderer to immediately stop and destroy all related resources. Examples include:
+* the user backs out of the video stream to return to the normal app UI
+* there was an unrelated error that requires immediate halt of the current ad unit
+* the app code has reached a custom timeout waiting for either the [`onFetchAdComplete`](#onfetchadcomplete) or [`onAdStarted`](#onadstarted) events
 
-If the request fails, the renderer will fire the `AD_ERROR` event.
+The `TruexAdRenderer` instance should not be used again after calling `stop` -- please remove all references to it afterwards.
 
-The parameters for this method call are:
+In contrast to `pause`, there is no way to resume the ad after `stop` is called.
 
-* `baseViewController`: The `viewController` that `TruexAdRenderer` should present its `ViewController` to. The `baseViewController` should cover the entire screen.
 
+#### `pause`
 
-#### stop
-
-    - (void)stop
+```objective-c
+    - (void)pause;
+```
 
-The `stop()` method is only called when the app needs the renderer to immediately stop and destroy all related resources.
+`pause` is required whenever the app needs to pause the true[X] unit (including all video/audio and countdowns).
```

Comment:
Super nit: consider rephrasing to "... all video, audio, and timers"

Reply:
I like that, will do